src/harness/common_strings.py

```diff
@@ -0,0 +1,18 @@
+"""Strings which are repeated throughout the test code.
+"""
+
+EXPECTED_SUCCESSFUL_REGISTRATION = (
+    'Registration was expected to succeed and failure to register almost '
+    'certainly indicates a configuration error. Please attempt registration of'
+    ' the same device using REG.11 and verify that it succeeds.'
```

Comment:
Would using the 'REG.11' in this message be misleading since this same message is used for different testcases?